README.md

```diff
@@ -298,6 +308,7 @@ Courses:
 ## Consensus
 
 - University of Cambridge [Distributed consensus revised](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-935.pdf), a great paper about Consenssus especially Paxos and Paxos-Related algorithms, by Heidi Howard
+- [Ark: A Real-World Consensus Implementation By Zardosht Kashef](https://arxiv.org/pdf/1407.4765.pdf)
```

Comment:
for papers, it's better to be formatted like this:
```
Papers:

- 2014, [Ark: A Real-World Consensus Implementation](https://arxiv.org/pdf/1407.4765.pdf), CoRR
```

BTW, the first one is a technical report, it's better to separate them into different sections.

Reply:
Agreed

Reply:
I mean "Distributed consensus revised" is a technical report, while "Ark: A Real-World Consensus Implementation" is a research paper. For this section, it's recommended to be formatted like this:

```

Technical Report:
- University of Cambridge [Distributed consensus revised](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-935.pdf), a great paper about Consenssus especially Paxos and Paxos-Related algorithms, by Heidi Howard

Paper:
 
- 2014, [Ark: A Real-World Consensus Implementation](https://arxiv.org/pdf/1407.4765.pdf), CoRR
```

Sorry for the misleading 😂

Reply:
OK now I get you. updated.